memory.cpp

```diff
@@ -253,36 +257,40 @@
 
             void InitHooks()
             {
-                // g_origMalloc = __malloc_hook;
-                // g_origFree = __free_hook;
-                // __malloc_hook = MallocHook;
-                // __free_hook = FreeHook;
+                #ifndef USE_MUSL_LIBC
+                    g_origMalloc = __malloc_hook;
+                    g_origFree = __free_hook;
+                    __malloc_hook = MallocHook;
+                    __free_hook = FreeHook;
+                #endif
             }
             void (* volatile __malloc_initialize_hook)() = InitHooks;
 
-            // void* MallocHook(size_t size, const void * context)
-            // {
-            //     if (pthread_getspecific(tls_key) || HasSEAInStack())
-            //         return g_origMalloc(size, context);
-            //     CRecursionScope scope;
-            //
-            //     __itt_heap_allocate_begin(g_heap, size, 0);
-            //     void* res = g_origMalloc(size, context);
-            //     __itt_heap_allocate_end(g_heap, &res, size, 0);
-            //
-            //     return res;
-            // }
-            //
-            // void FreeHook(void* ptr, const void* context)
-            // {
-            //     if (pthread_getspecific(tls_key) || HasSEAInStack())
-            //         return g_origFree(ptr, context);
-            //     CRecursionScope scope;
-            //
-            //     __itt_heap_free_begin(g_heap, ptr);
-            //     g_origFree(ptr, context);
-            //     __itt_heap_free_end(g_heap, ptr);
-            // }
+            #ifndef USE_MUSL_LIBC
+                void* MallocHook(size_t size, const void * context)
```

Comment:
Why was this commented out before and now it is not. If USE_MUSL_LIBC is not defined should this code really be executed?